test/python/topology/test2_submission_result.py

```diff
@@ -12,6 +12,8 @@
 class TestSubmissionResult(unittest.TestCase):
     def setUp(self):
         Tester.setup_distributed(self)
+        self.username = os.getenv("STREAMS_USERNAME", "streamsadmin")
```

Comment:
Seems to be new fields added with no added use of them?

Reply:
They are used on line 28 of the same file. These fields were previously set in `Tester`, but are now set in the `setUp` method.